spec/models/tslr_claim_spec.rb

```diff
@@ -365,7 +365,26 @@
       let(:tslr_claim) { create(:tslr_claim, :eligible_and_submittable) }
 
       it "sets submitted_at to now" do
-        expect(tslr_claim.submitted_at).to eq Time.zone.now
+        expect(tslr_claim.reload.submitted_at).to eq Time.zone.now
+      end
+
+      it "generates a reference" do
+        expect(tslr_claim.reload.reference).to_not eq nil
```

Comment:
I don't think these should need reloading because they're already saved?

Reply:
Ah, you're right. In a different permutation I'd run `update_columns`, which didn't update the attributes in place, but I've since binned that off